libs/core/state/src/operation/entity/selectors.ts

```diff
@@ -1,9 +1,9 @@
-import type { EntitySelectors } from '@ngrx/entity/src/models';
 import {
   createSelector,
   defaultMemoize,
   MemoizedSelector,
 } from '@ngrx/store';
+import type { EntitySelectors } from 'node_modules/@ngrx/entity/src/models';
```

Comment:
This look very wrong.

Reply:
Well really we shouldn't ever be importing anything from `src`. The reason why this change is required is because `bundler` respects the packages `exports` field, which obviously doesn't list `src` as an export. There *is* a fairly complex way of extracting this type that doesn't rely on an internal import, should I do that?

Reply:
Yes, It's a PITA to read I'd imagine, but it's worth not having to pull a type directly from a node_modules path.

Reply:
okay done